### DISCUSSION TITLE
Embed widget in form tab

### DISCUSSION THREAD

**Discussion by ashleyhood** [filamentphp/filament#5022] [Help]
I have a `ClientResource` and in `app/Filament/Resources/ClientResource/Pages/ViewClient` my `getFormSchema` method has a `Tabs` component. In one of the tabs, I want to render a widget (saad/filament-fullcalendar). What is the recommended way of rendering widgets in a `View` component?

I have tried adding `@livewire(\Livewire\Livewire::getAlias($widget), $data, key($widget))` to the `View` but when I switch away from the tab with the widget and then switch back to it, I get the error `initialData is null` in the browser console and the widget does not render.

If I set the `key` to `key(time())` it works but that seems like I'm doing something wrong.

**Comment by fuzzyhope1502** [comment]
I'm also curious about this. I have some tables as widgets for my user and displaying data, and I want to just nest them in tabs but doesn't appear to be much docs wise on it. Plenty of information on the form builder, but not much on getting it working with table/admin panel stuff.

**Reply by tonypartridge** [reply]
FYI you can use Livewire::make() in V3/4
